pkgs/cronet_http/lib/src/cronet_client.dart

```diff
@@ -28,20 +28,39 @@ class _StreamedResponseWithUrl extends StreamedResponse
       super.reasonPhrase});
 }
 
+class UrlResponseInfo {
```

Comment:
I think that this should be private. Also, would it make sense to just add these add fields to `CronetStreamedResponse`?